jetbrains-core/src/software/aws/toolkits/jetbrains/services/dynamo/Attributes.kt

```diff
@@ -0,0 +1,93 @@
+// Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.services.dynamo
+
+import software.amazon.awssdk.services.dynamodb.model.AttributeValue
+import java.util.Base64
+
+typealias SearchResults = List<Map<String, DynamoAttribute<*>>>
+
+sealed class DynamoAttribute<T>(val value: T) {
+    abstract val dataType: String
+
+    open fun stringRepresentation() = value.toString()
+}
+
+class StringAttribute(value: String) : DynamoAttribute<String>(value) {
+    override val dataType: String = "S"
+
+    override fun stringRepresentation(): String = "\"$value\""
+}
+
+class BooleanAttribute(value: Boolean) : DynamoAttribute<Boolean>(value) {
+    override val dataType: String = "BOOL"
+}
+
+class NumberAttribute(value: String) : DynamoAttribute<String>(value) {
+    override val dataType: String = "N"
+}
+
+class BinaryAttribute(value: ByteArray) : DynamoAttribute<ByteArray>(value) {
+    override val dataType: String = "B"
+
+    override fun stringRepresentation(): String = Base64.getEncoder().encodeToString(value)
+}
+
+object NullAttribute : DynamoAttribute<Boolean>(/*Dynamo always expects the NUL field to contain true */true) {
+    override val dataType: String = "NUL"
+
+    override fun stringRepresentation(): String = "<null>"
+}
+
+class StringSetAttribute(value: List<String>) : DynamoAttribute<List<String>>(value) {
+    override val dataType: String = "SS"
+
+    override fun stringRepresentation(): String = value.joinToString(prefix = "{", postfix = "}") {
+        "\"$it\""
```

Comment:
Do we need to do any kind of escaping or anything?

Reply:
The console does, but I think it makes it more misleading.

They Show `"Hello World"` as `\"Hello World\"` even though it is not escaped when using the raw data from the API.